app/models/user.rb

```diff
@@ -1668,6 +1668,15 @@ def sidebar_categories_ids
     categories_ids
   end
 
+  def sidebar_tags
+    return custom_sidebar_tags if custom_sidebar_tags.present?
+    if SiteSetting.default_sidebar_tags.present?
+      tag_names = SiteSetting.default_sidebar_tags.split("|") - DiscourseTagging.hidden_tag_names(guardian)
+      return Tag.where(name: tag_names)
+    end
+    []
```

Comment:
```suggestion
    Tag.none
```

Let's introduce a consistent type in our return value here.